public/app/features/dashboard-scene/sharing/ShareButton/share-snapshot/ShareSnapshot.tsx

```diff
@@ -109,7 +110,7 @@ function ShareSnapshotRenderer({ model }: SceneComponentProps<ShareSnapshot>) {
                 />
               )
             )}
-            <TextLink icon="external-link-alt" href="/dashboard/snapshots" external>
+            <TextLink icon="external-link-alt" href={`${config.appSubUrl || ''}/dashboard/snapshots`}>
```

Comment:
Removing `external` will make the link open in the same tab, and also will treat this as a "internal" Link and will call this `stripBaseFromURL` that has some logic around appSubUrl